contents/copy/speakers/guide.md

```diff
@@ -41,18 +41,71 @@ If you're travelling to the venue by taxi, most taxi drivers should recognize th
 
 ## Bookings & Reimbursements
 
-For most of you, everything is already booked. If you do need reimbursement, please contact Simone! She'll help you figure everything out.
+For most of you, everything is already booked. If you do need reimbursement, please contact Simone ([simone@jsconf.eu](mailto:simone@jsconf.eu))! She'll help you figure everything out. If your company has offered to pay: Awesome, thanks! We’d be happy to list them as a supporter on our website, so just let us know if that’s the case.
```

Comment:
i feel like we removed the last sentence for a reason... but i can't remember

Reply:
Let's leave it in – for CSSconf it makes sense, if we receive any requests to add companies to our sponsor page, we'll be able to handle it :)